smartMeter/ngsild-payloads/consumer.json

```diff
@@ -0,0 +1,128 @@
+{
+    "id": "urn:ngsi-ld:Consumer:01",
+    "type": "Consumer",
+    "waterConsumption": {
+        "type": "Property",
+        "value": 1500,
+        "unitCode": "LTR",
+        "observedAt": "2020-06-12T15:11:36.158Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:smartMeter01"
+        }
+    },
+    "minFlow": {
+        "type": "Property",
+        "value": 1,
+        "unitCode": "MQS",
```

Comment:
The actual unit of the smart meter is "pulse per hour". Suggested CEFACT would be1M (reciprocal hour). 
How many litre correspond to 1 pulse depends on the water meter and pipe DIN size. From Dielh documentarian, it may be 1l or 10l per pulse. 
I understand from the examples we got that we are in a case of 1l/pulse thus the unit code to be used is "E32" (Litre per hour)

Reply:
Ok